python/train.py

```diff
@@ -91,6 +91,8 @@
   parser.add_argument('-intermediate-loss-scale', type=float, help='Loss factor scale for intermediate head', required=False)
   parser.add_argument('-intermediate-distill-scale', type=float, help='Distill factor scale for intermediate head', required=False)
 
+  parser.add_argument('-disable-vtimeloss', help='Disable vtimeloss for training', required=False, action='store_true')
```

Comment:
Alternative:
```suggestion
  parser.add_argument('-disable-vtimeloss', help='Disable vtimeloss for training', required=False, dest='use_vtimeloss', action='store_false')
```
then don't need the `not` later.

Not sure this is actually any better though but figured I'd give the option as the `dest` argument is easily overlooked.